### DISCUSSION TITLE
Add builds to plugins repositories

### DISCUSSION THREAD

**Discussion by adam3278** [GrapesJS/grapesjs#5881] [Ideas]
Hi,
Is there any possibility, that you would add js builds to GrapesJS officially maintained plugins? The plugins are available to use with TypeScript only (as far as I see) and only working builds are available as remote resources. It's very convenient to clone git repository into our own project as git submodule and update it simply pulling it (especially via VS Code). Compiling it every repo commit for many plugins is a pain in the neck. Thanks in advance.

**Comment by artf** [comment]
Builds shouldn't be inside repositories, you should download and use plugins via a package manager like npm, like everybody does.

> The plugins are available to use with TypeScript only

What do you mean? You can import the plugin in a regular JS file

**Reply by dababyismaybe** [reply]
Would you mind explaining how to import plugins with a JS file? I am a bit new to this and wanted to make a cool site!

**Reply by artf** [reply]
https://grapesjs.com/docs/modules/Plugins.html

**Reply by dababyismaybe** [reply]
Yes I have seen this many times... I don't know how to use the dang thing :(

I just want to add a bunch of plugins in order to make a fun site but I don't know:

which files to edit and also where to add the source files for the plugins etc. I have been trying everything but I am pretty new to github so I have been VERY confused.
